app.rb

```diff
@@ -7,9 +8,9 @@ class ComplianceData < Sinatra::Base
 
   config_file 'credentials.yml'
 
-  configure :production do
-    use Rack::Auth::Basic do |username, password|
-      username == settings.site_user && password == settings.site_password
+  helpers do
+    def authed?
+      session[:authenticated]
```

Comment:
Maybe just use `!!session[:user_email]`? Then there's no need for `session[:authenticated]`.